ui/lib/sync/addon/components/secrets/landing-cta.hbs

```diff
@@ -29,17 +28,20 @@
     />
   {{/if}}
 </div>
-<div class="box is-fullwidth is-shadowless">
-  <p>
-    <b>Step 1:</b>
-    Create a destination, and set up the connection details to allow Vault access.
-  </p>
-  <p>
-    <b>Step 2:</b>
-    Select secrets from your KV v2 engine and sync secrets to your destination.
-  </p>
-</div>
 
-<div class="box is-fullwidth is-shadowless">
-  {{!-- <img src={{img-path "~/some-photo-location.png"}} alt="Secrets sync diagram" /> --}}
+<div class="is-flex-row">
+  <div>
+    <img src={{img-path "~/sync-landing-1.png"}} alt="Secrets sync destinations diagram" />
```

Comment:
We may be able to leverage the copy that's already here as an image description. eg. 
```suggestion
    <img src={{img-path "~/sync-landing-1.png"}} alt="Secrets sync destinations diagram" aria-describedby="sync-step1" />
    <p id="sync-step1">...</p>
```

Reply:
Good suggestion I will add that for both images. :accessibility: